analysis/src/Packages.ml

```diff
@@ -156,6 +156,7 @@ let newBsPackage ~rootPath =
                          | ["RescriptCore"] -> true
                          | _ -> false)
                   |> Option.is_some
+                  || fst rescriptVersion >= 12
```

Comment:
You know what. This is in the compiler repo, right. So it'll always apply here. We can remove this entire code path and just always return what's returned by this branch here. This is because the other code deals with older ReScript versions, but we know the equivalent of Core will always be available through the stdlib.

Reply:
I guess it'll update the test output. Maybe that's better to do in a separate PR. So keep this and we streamline that separately.

Reply:
We can remove in next PR.